src/edu/cs505/g5/ObserverBuzzer.java

```diff
@@ -0,0 +1,21 @@
+package edu.ccsu.cs505.g5;
+
+/**
+ * This class implements the interface ObserverDHTSensor
+ * Observer State is an attribute
+ * @author CS505-Group5
+ */
+public class ObserverBuzzer implements ObserverDHTSensor {
+
+	public int observerState;

```

Comment:
In general don't make any public attributes.  Make the actual attributes private or protected and add accessors (i.e. getters/setters)

Reply:
Note you want to have your concrete Observers have a reference to the concrete class they are interested in.  The notification method just lets it know that what it was listening to changed, then it uses its reference to the concrete observer to look at the details of what changed.

Reply:
modified.